odoo/addons/base/models/res_config.py

```diff
@@ -437,14 +441,15 @@ def _get_classified_fields(self):
                 {   'default': [('default_foo', 'model', 'foo'), ...],
                     'group':   [('group_bar', [browse_group], browse_implied_group), ...],
                     'module':  [('module_baz', browse_module), ...],
+                    'config':  [('plop', config_parameter), ...],
```

Comment:
Please use the parameter name from the example above, that makes it less confusing:
```python
    'config': [('config_qux', 'a.global.parameter'), ...],
```
Note that I had to fix this note, as I got it wrong the first time :wink: